torchvision/transforms/v2/functional/_misc.py

```diff
@@ -72,6 +79,52 @@ def normalize_video(video: torch.Tensor, mean: list[float], std: list[float], in
     return normalize_image(video, mean, std, inplace=inplace)
 
 
+def _normalize_cvcuda(
+    image: "cvcuda.Tensor",
+    mean: list[float],
+    std: list[float],
+    inplace: bool = False,
+) -> "cvcuda.Tensor":
+    cvcuda = _import_cvcuda()
+    if inplace:
+        raise ValueError("Inplace normalization is not supported for CVCUDA.")
+
+    # CV-CUDA supports signed int and float tensors
+    # torchvision only supports uint and float, right now CV-CUDA doesnt expose float16, so only check 32
+    # in the future add float16 once exposed in CV-CUDA
+    if not (image.dtype == cvcuda.Type.F32):
+        raise ValueError(f"Input tensor should be a float tensor. Got {image.dtype}.")
+
+    channels = image.shape[3]
+    if isinstance(mean, float | int):
+        mean = [mean] * channels
+    elif len(mean) != channels:
+        raise ValueError(f"Mean should have {channels} elements. Got {len(mean)}.")
+    if isinstance(std, float | int):
+        std = [std] * channels
+    elif len(std) != channels:
+        raise ValueError(f"Std should have {channels} elements. Got {len(std)}.")
+
+    # CV-CUDA requires float32 tensors for the mean/std parameters
+    # at small batchs, this is costly relative to normalize operation
+    # if CV-CUDA is known to be a backend, could optimize this
+    # For Normalize class:
+    # by creating tensors at class initialization time
+    # For functional API:
+    # by storing cached tensors in helper function with functools.lru_cache (would it even be worth it?)
+    # Since CV-CUDA is 1) not default backend, 2) only strictly faster at large batch size, ignore
+    mt = torch.as_tensor(mean, dtype=torch.float32).reshape(1, 1, 1, channels).cuda()
+    st = torch.as_tensor(std, dtype=torch.float32).reshape(1, 1, 1, channels).cuda()
+    mean_cv = cvcuda.as_tensor(mt, cvcuda.TensorLayout.NHWC)
+    std_cv = cvcuda.as_tensor(st, cvcuda.TensorLayout.NHWC)
+
+    return cvcuda.normalize(image, base=mean_cv, scale=std_cv, flags=cvcuda.NormalizeFlags.SCALE_IS_STDDEV)
+
+
+if CVCUDA_AVAILABLE:
+    _normalize_cvcuda_registered = _register_kernel_internal(normalize, _import_cvcuda().Tensor)(_normalize_cvcuda)
```

Comment:
We should be able to remove `_normalize_cvcuda_registered =` here and keep only `_register_kernel_internal(normalize, _import_cvcuda().Tensor)(_normalize_cvcuda)`.